buildsystem/modules/FindPython.cmake

```diff
@@ -69,22 +74,32 @@ find_package(Python3 ${PYTHON_USED_VERSION} ${need_exact_version} COMPONENTS Int
 math(EXPR BIT_SHIFT_HEX "${Python3_VERSION_MAJOR} << 24 | ${Python3_VERSION_MINOR} << 16" OUTPUT_FORMAT HEXADECIMAL)
 set(PYTHON_MIN_VERSION_HEX "${BIT_SHIFT_HEX}")
 
+
 # there's a static_assert that tests the Python version.
 # that way, we verify the interpreter and the library version.
 # (the interpreter provided us the library location)
 try_compile(PYTHON_TEST_RESULT
 	"${CMAKE_BINARY_DIR}"
 	SOURCES "${CMAKE_CURRENT_LIST_DIR}/FindPython_test.cpp"
 	LINK_LIBRARIES Python3::Python
-	CXX_STANDARD 17
-	COMPILE_DEFINITIONS "-DTARGET_VERSION=${PYTHON_MIN_VERSION_HEX}"
+	CXX_STANDARD ${CMAKE_CXX_STANDARD}
+    COMPILE_DEFINITIONS "-DTARGET_VERSION=${PYTHON_MIN_VERSION_HEX}"
```

Comment:
indent accidently switched to spaces?

Reply:
damn you, vim. if anyone knows a good way like emacs' `.dir-locals.el` file to configure project specific indentation settings, please tell me...